x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_forms/eventing/checkbox.tsx

```diff
@@ -27,7 +27,11 @@ export const EventingCheckbox: React.FC<{
     (event: React.ChangeEvent<HTMLInputElement>) => {
       if (policyDetailsConfig) {
         const newPayload = clone(policyDetailsConfig);
-        newPayload[os].eventing[protectionField] = event.target.checked;
+        if (os === OS.linux || os === OS.mac) {
+          newPayload[os].events.process = event.target.checked;
```

Comment:
there will eventually be more than just `process` - we can address in later PRs where we add those forms

Reply:
Yeah. This is where i had trouble earlier with the types and had to include the `if` branches here. This entire callback will need to be refactored when we add all the other types back to the policy model and start to work the UI for it.